TRD: Calibration update and additonal macros

f3sch opened the pull request:
I updated the README and added two macros:
- plotVdriftExB.C plots the VDrift and ExB values in the ccdb for a given duration for each chamber
- manualCalibFit.C performs the calibration fits for VDrift and ExB manually

f3sch committed:
TRD: calibration: README update

Just added some newlines (easier to edit) and slightly altered formatting.

f3sch committed:
TRD: calibration: added instructions and macro for manual calibration fits

f3sch committed:
TRD: calibration: Added macro to plot VDrift and ExB values

f3sch committed:
TRD: calibration: README formatting

alibuild commented:
Error while checking build/O2/o2-dataflow for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-20 05:21:
```
## sw/BUILD/O2-latest/log
253/429 Test #254: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    2.57 sec
/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/bin/rootcling -v0 "--lib-list-prefix=/sw/BUILD/cd230f868a843f5aff449e87bbc3fbb5cfe263c8/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/sw/BUILD/cd230f868a843f5aff449e87bbc3fbb5cfe263c8/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -I"/sw/slc7_x86-64/ms_gsl/4.0.0-10/include" -I"/sw/slc7_x86-64/Vc/1.4.1-93/include" -I"/sw/slc7_x86-64/FairRoot/v18.4.8-21/include" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-2/include/fairmq" -I"/sw/slc7_x86-64/asio/v1.19.1-19/include" -I"/sw/slc7_x86-64/FairLogger/v1.11.1-3/include" -I"/sw/slc7_x86-64/fmt/8.1.1-9/include" -I"/sw/slc7_x86-64/VMC/v2-0-22/include/vmc" -I"/sw/slc7_x86-64/boost/v1.75.0-81/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Detectors/TRD/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/common/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Constants/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/GPU/Common" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Logger/include" -I"/sw/slc7_x86-64/fmt/8.1.1-9/include" -I"/sw/slc7_x86-64/FairLogger/v1.11.1-3/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/include" -I"/sw/slc7_x86-64/FairRoot/v18.4.8-21/include" -I"/sw/slc7_x86-64/VMC/v2-0-22/include/vmc" -I"/sw/slc7_x86-64/arrow/v9.0.0-alice1-5/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/MathUtils/include" -I"/sw/slc7_x86-64/Vc/1.4.1-93/include" -I"/sw/slc7_x86-64/ms_gsl/4.0.0-10/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/simulation/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Detectors/Common/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/GPU/Utils" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Headers/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/MemoryResources/include" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-2/include/fairmq" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-2/include" -I"/sw/slc7_x86-64/boost/v1.75.0-81/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Utilities/rANS/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Core/include" -I"/sw/slc7_x86-64/Configuration/v2.6.3-34/include" -I"/sw/slc7_x86-64/libInfoLogger/v2.5.1-5/include" -I"/sw/slc7_x86-64/Monitoring/v3.13.2-1/include" -I"/sw/slc7_x86-64/curl/7.70.0-53/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/CCDB/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Utils/include" -I"/sw/slc7_x86-64/libjalienO2/0.1.3-24/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Foundation/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Foundation/3rdparty/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Utilities/PCG/include" -I"/sw/slc7_x86-64/RapidJSON/v1.1.0-alice2-37/include" -I"/sw/slc7_x86-64/libuv/v1.40.0-27/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/Base/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Reconstruction/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Field/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Parameters/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Types/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/SimConfig/include" -I"/sw/slc7_x86-64/TBB/v2021.5.0-27/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/etc/" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/etc//cling" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/etc//cling/plugins/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-25/include/" -D__ACLIC__  "/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/sw/BUILD/cd230f868a843f5aff449e87bbc3fbb5cfe263c8/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 429
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_O2_o2-dataflow/pretty.html).

martenole commented on Detectors/TRD/calibration/README.md:
```diff
@@ -6,19 +6,53 @@
 
 To run the calibration one can start the following workflow:
+
     o2-trd-global-tracking -b --disable-root-output | o2-calibration-trd-vdrift-exb -b --calib-vdexb-calibration '--tf-per-slot 5 --min-entries 50000 --max-delay 90000'
```
Actually this is wrong I think (mistake made by me). Can you please change this line to
`o2-trd-global-tracking -b --disable-root-output --enable-trackbased-calib | o2-calibration-trd-vdrift-exb -b --calib-vdexb-calibration '--tf-per-slot 5 --min-entries 50000 --max-delay 90000'`?

alibuild commented:
Error while checking build/AliceO2/O2/o2/macOS-arm for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-19 17:22:
```
## sw/BUILD/O2-latest/log
252/451 Test #254: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    1.56 sec
/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/bin/rootcling -v0 "--lib-list-prefix=/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/BUILD/14c7c3227835341e2912763dd76c66f36f25085b/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/BUILD/14c7c3227835341e2912763dd76c66f36f25085b/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -isysroot "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/pythia/v8304-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ms_gsl/4.0.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/HepMC3/3.2.5-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/Vc/1.4.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairRoot/v18.4.8-local5/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairMQ/v1.4.55-local1/include/fairmq" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/asio/v1.19.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairLogger/v1.11.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/fmt/8.1.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/GEANT3/v4-1-local2/include/TGeant3" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/GEANT4_VMC/v6-1-p1-local2/include/g4root" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/GEANT4_VMC/v6-1-p1-local2/include/geant4vmc" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/GEANT4/v11.0.2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/GEANT4/v11.0.2-local1/include/Geant4" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/VMC/v2-0-local2/include/vmc" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/boost/v1.75.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/JAliEn-ROOT/0.6.6-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Detectors/TRD/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/common/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Constants/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/GPU/Common" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Logger/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/fmt/8.1.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairLogger/v1.11.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairRoot/v18.4.8-local5/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/VMC/v2-0-local2/include/vmc" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/arrow/v9.0.0-alice1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/MathUtils/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/Vc/1.4.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ms_gsl/4.0.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/simulation/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Detectors/Common/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/GPU/Utils" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Headers/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/MemoryResources/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairMQ/v1.4.55-local1/include/fairmq" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/FairMQ/v1.4.55-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/boost/v1.75.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Utilities/rANS/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Core/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/Configuration/v2.6.3-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/libInfoLogger/v2.5.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/Monitoring/v3.13.2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/curl/7.70.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/CCDB/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Utils/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/libjalienO2/0.1.3-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Foundation/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Foundation/3rdparty/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Utilities/PCG/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/RapidJSON/v1.1.0-alice2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/libuv/v1.40.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/Base/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Reconstruction/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Field/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Parameters/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Types/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Common/SimConfig/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/TBB/v2021.5.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/etc/" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/etc//cling" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/etc//cling/plugins/include" -I"/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/osx_arm64/ROOT/v6-26-04-patches-alice1-local2/include/" -D__ACLIC__  "/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/BUILD/14c7c3227835341e2912763dd76c66f36f25085b/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 451
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_AliceO2_O2_o2_macOS-arm/pretty.html).

martenole reviewed (dismissed):
Thanks a lot, this is quite helpful. I think it should also be possible to retrieve for a given run number the GRP object from the CCDB. But then one needs to use not only the time, since multiple runs can be ongoing in parallel. @shahor02 do you know how one can request the GRP object from a macro using the CCDB manager? Is this supported?
Then we could get the run start and stop time from that object and don't need to get it manually from bookkeeping.

shahor02 commented:
> Thanks a lot, this is quite helpful. I think it should also be possible to retrieve for a given run number the GRP object from the CCDB. But then one needs to use not only the time, since multiple runs can be ongoing in parallel. @shahor02 do you know how one can request the GRP object from a macro using the CCDB manager? Is this supported? Then we could get the run start and stop time from that object and don't need to get it manually from bookkeeping.

Hi, one should provide the run number via metadata:
```
  auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
  ...
  std::map<std::string, std::string> md;
  md["runNumber"] = std::to_string(runNumber);
  const auto* grp = ccdbMgr.getSpecific<o2::parameters::GRPECSObject>("GLO/Config/GRPECS", timestamp, md);
```

alibuild commented:
Error while checking build/O2/o2-dataflow-cs8 for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-19 19:48:
```
## sw/BUILD/O2-latest/log
254/429 Test #254: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    7.27 sec
/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/bin/rootcling -v0 "--lib-list-prefix=/sw/BUILD/9135d0124739e8a3caad51cf5b85da2e51c113fd/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/sw/BUILD/9135d0124739e8a3caad51cf5b85da2e51c113fd/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -I"/sw/slc8_x86-64/ms_gsl/4.0.0-12/include" -I"/sw/slc8_x86-64/Vc/1.4.1-29/include" -I"/sw/slc8_x86-64/FairRoot/v18.4.8-38/include" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-3/include/fairmq" -I"/sw/slc8_x86-64/asio/v1.19.1-23/include" -I"/sw/slc8_x86-64/FairLogger/v1.11.1-4/include" -I"/sw/slc8_x86-64/fmt/8.1.1-10/include" -I"/sw/slc8_x86-64/VMC/v2-0-27/include/vmc" -I"/sw/slc8_x86-64/boost/v1.75.0-53/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Detectors/TRD/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/common/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Constants/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/GPU/Common" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Logger/include" -I"/sw/slc8_x86-64/fmt/8.1.1-10/include" -I"/sw/slc8_x86-64/FairLogger/v1.11.1-4/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/include" -I"/sw/slc8_x86-64/FairRoot/v18.4.8-38/include" -I"/sw/slc8_x86-64/VMC/v2-0-27/include/vmc" -I"/sw/slc8_x86-64/arrow/v9.0.0-alice1-6/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/MathUtils/include" -I"/sw/slc8_x86-64/Vc/1.4.1-29/include" -I"/sw/slc8_x86-64/ms_gsl/4.0.0-12/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/simulation/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Detectors/Common/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/GPU/Utils" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Headers/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/MemoryResources/include" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-3/include/fairmq" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-3/include" -I"/sw/slc8_x86-64/boost/v1.75.0-53/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Utilities/rANS/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Core/include" -I"/sw/slc8_x86-64/Configuration/v2.6.3-45/include" -I"/sw/slc8_x86-64/libInfoLogger/v2.5.1-8/include" -I"/sw/slc8_x86-64/Monitoring/v3.13.2-1/include" -I"/sw/slc8_x86-64/curl/7.70.0-4/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/CCDB/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Utils/include" -I"/sw/slc8_x86-64/libjalienO2/0.1.3-26/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Foundation/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Foundation/3rdparty/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Utilities/PCG/include" -I"/sw/slc8_x86-64/RapidJSON/v1.1.0-alice2-36/include" -I"/sw/slc8_x86-64/libuv/v1.40.0-29/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/Base/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Reconstruction/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Field/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Parameters/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Types/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/SimConfig/include" -I"/sw/slc8_x86-64/TBB/v2021.5.0-21/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/etc/" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/etc//cling" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/etc//cling/plugins/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-23/include/" -D__ACLIC__  "/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/sw/BUILD/9135d0124739e8a3caad51cf5b85da2e51c113fd/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 429
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_O2_o2-dataflow-cs8/pretty.html).

martenole commented on Detectors/TRD/calibration/macros/CMakeLists.txt:
```diff
@@ -32,8 +32,19 @@ o2_add_test_root_macro(readTRDVoltages.C
                       LABELS trd COMPILE_ONLY)
 
+o2_add_test_root_macro(manualCalibFit.C
+                      PUBLIC_LINK_LIBRARIES O2::DataFormatsTRD
```
I think you need to add `O2::TRDCalibration` here as well. The CI tests are failing with `/System/Volumes/Data/build/nomad/alloc/627e7c3c-41b1-6bbb-b5b0-3fab21115acf/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found`

martenole commented:
> Hi, one should provide the run number via metadata:
> 
> ```
>   auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
>   ...
>   std::map<std::string, std::string> md;
>   md["runNumber"] = std::to_string(runNumber);
>   const auto* grp = ccdbMgr.getSpecific<o2::parameters::GRPECSObject>("GLO/Config/GRPECS", timestamp, md);
> ```

Thanks a lot Ruben! @f3sch could you add this method such that one does not need to specify time stamps anymore? You can take a look at `DataFormats/Parameters/include/DataFormatsParameters/GRPECSObject.h`. With `grp->getTimeStart()` and `grp->getTimeEnd()` you can get beginning and end time of the run.

alibuild commented:
Error while checking build/AliceO2/O2/o2/macOS for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-19 21:12:
```
## sw/BUILD/O2-latest/log
249/451 Test #254: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    3.67 sec
/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/bin/rootcling -v0 "--lib-list-prefix=/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/BUILD/8f9f000bcc216b347541af59dce200b77bc4c428/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/BUILD/8f9f000bcc216b347541af59dce200b77bc4c428/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -isysroot "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/pythia/v8304-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ms_gsl/4.0.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/HepMC3/3.2.5-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/Vc/1.4.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairRoot/v18.4.8-local5/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairMQ/v1.4.55-local1/include/fairmq" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/asio/v1.19.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairLogger/v1.11.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/fmt/8.1.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/GEANT3/v4-1-local2/include/TGeant3" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/GEANT4_VMC/v6-1-p1-local2/include/g4root" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/GEANT4_VMC/v6-1-p1-local2/include/geant4vmc" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/GEANT4/v11.0.2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/GEANT4/v11.0.2-local1/include/Geant4" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/VMC/v2-0-local2/include/vmc" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/boost/v1.75.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/JAliEn-ROOT/0.6.6-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Detectors/TRD/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/common/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Constants/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/GPU/Common" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Logger/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/fmt/8.1.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairLogger/v1.11.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairRoot/v18.4.8-local5/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/VMC/v2-0-local2/include/vmc" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/arrow/v9.0.0-alice1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/MathUtils/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/Vc/1.4.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ms_gsl/4.0.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/simulation/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Detectors/Common/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/GPU/Utils" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Headers/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/MemoryResources/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairMQ/v1.4.55-local1/include/fairmq" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/FairMQ/v1.4.55-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/boost/v1.75.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Utilities/rANS/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Core/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/Configuration/v2.6.3-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/libInfoLogger/v2.5.1-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/Monitoring/v3.13.2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/curl/7.70.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/CCDB/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Utils/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/libjalienO2/0.1.3-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Foundation/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Framework/Foundation/3rdparty/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Utilities/PCG/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/RapidJSON/v1.1.0-alice2-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/libuv/v1.40.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/Base/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Reconstruction/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Field/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/DataFormats/Parameters/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/Types/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Common/SimConfig/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/TBB/v2021.5.0-local1/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/etc/" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/etc//cling" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/etc//cling/plugins/include" -I"/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/osx_x86-64/ROOT/v6-26-04-patches-alice1-local2/include/" -D__ACLIC__  "/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/BUILD/8f9f000bcc216b347541af59dce200b77bc4c428/O2/compiled_macros//System/Volumes/Data/build/nomad/alloc/f7552d80-4750-bbd0-a1cd-d8b3785f2966/ci/local/o2/sw/SOURCES/O2/9863/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 451
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_AliceO2_O2_o2_macOS/pretty.html).

alibuild commented:
Error while checking build/O2/o2 for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-19 21:16:
```
## sw/BUILD/O2-latest/log
263/452 Test #255: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    5.61 sec
/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/bin/rootcling -v0 "--lib-list-prefix=/sw/BUILD/b5311929c094bc747c5667d02d988987648c6c1d/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/sw/BUILD/b5311929c094bc747c5667d02d988987648c6c1d/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -I"/sw/slc7_x86-64/pythia/v8304-45/include" -I"/sw/slc7_x86-64/ms_gsl/4.0.0-6/include" -I"/sw/slc7_x86-64/HepMC3/3.2.5-20/include" -I"/sw/slc7_x86-64/Vc/1.4.1-87/include" -I"/sw/slc7_x86-64/FairRoot/v18.4.8-20/include" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-1/include/fairmq" -I"/sw/slc7_x86-64/asio/v1.19.1-15/include" -I"/sw/slc7_x86-64/FairLogger/v1.11.1-1/include" -I"/sw/slc7_x86-64/fmt/8.1.1-5/include" -I"/sw/slc7_x86-64/GEANT3/v4-1-14/include/TGeant3" -I"/sw/slc7_x86-64/GEANT4_VMC/v6-1-p1-3/include/g4root" -I"/sw/slc7_x86-64/GEANT4_VMC/v6-1-p1-3/include/geant4vmc" -I"/sw/slc7_x86-64/GEANT4/v11.0.2-1/include" -I"/sw/slc7_x86-64/GEANT4/v11.0.2-1/include/Geant4" -I"/sw/slc7_x86-64/VMC/v2-0-21/include/vmc" -I"/sw/slc7_x86-64/boost/v1.75.0-79/include" -I"/sw/slc7_x86-64/JAliEn-ROOT/0.6.6-140/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Detectors/TRD/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/common/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Constants/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/GPU/Common" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Logger/include" -I"/sw/slc7_x86-64/fmt/8.1.1-5/include" -I"/sw/slc7_x86-64/FairLogger/v1.11.1-1/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/include" -I"/sw/slc7_x86-64/FairRoot/v18.4.8-20/include" -I"/sw/slc7_x86-64/VMC/v2-0-21/include/vmc" -I"/sw/slc7_x86-64/arrow/v9.0.0-alice1-4/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/MathUtils/include" -I"/sw/slc7_x86-64/Vc/1.4.1-87/include" -I"/sw/slc7_x86-64/ms_gsl/4.0.0-6/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/simulation/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Detectors/Common/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/GPU/Utils" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Headers/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/MemoryResources/include" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-1/include/fairmq" -I"/sw/slc7_x86-64/FairMQ/v1.4.55-1/include" -I"/sw/slc7_x86-64/boost/v1.75.0-79/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Utilities/rANS/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Core/include" -I"/sw/slc7_x86-64/Configuration/v2.6.3-32/include" -I"/sw/slc7_x86-64/libInfoLogger/v2.5.1-4/include" -I"/sw/slc7_x86-64/Monitoring/v3.13.2-2/include" -I"/sw/slc7_x86-64/curl/7.70.0-48/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/CCDB/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Utils/include" -I"/sw/slc7_x86-64/libjalienO2/0.1.3-20/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Foundation/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Framework/Foundation/3rdparty/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Utilities/PCG/include" -I"/sw/slc7_x86-64/RapidJSON/v1.1.0-alice2-36/include" -I"/sw/slc7_x86-64/libuv/v1.40.0-23/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/Base/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Reconstruction/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Field/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/DataFormats/Parameters/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/Types/include" -I"/sw/SOURCES/O2/9863-slc7_x86-64/0/Common/SimConfig/include" -I"/sw/slc7_x86-64/TBB/v2021.5.0-21/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/etc/" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/etc//cling" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/etc//cling/plugins/include" -I"/sw/slc7_x86-64/ROOT/v6-26-04-patches-alice1-24/include/" -D__ACLIC__  "/sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/sw/BUILD/b5311929c094bc747c5667d02d988987648c6c1d/O2/compiled_macros//sw/SOURCES/O2/9863-slc7_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 452
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_O2_o2/pretty.html).

alibuild commented:
Error while checking build/O2/o2-cs8 for 521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5 at 2022-09-19 21:26:
```
## sw/BUILD/O2-latest/log
252/452 Test #255: Detectors/TRD/calibration/macros/manualCalibFit.C_compiled ...........................***Failed    4.68 sec
/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C:32:10: fatal error: 'TRDCalibration/CalibratorVdExB.h' file not found
Error in <ACLiC>: Executing '/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/bin/rootcling -v0 "--lib-list-prefix=/sw/BUILD/921c390b408038575f5d92a94a5e177e9bf966f0/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_map" -f "/sw/BUILD/921c390b408038575f5d92a94a5e177e9bf966f0/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_dict.cxx" -I$ROOTSYS/include -I"/sw/slc8_x86-64/pythia/v8304-31/include" -I"/sw/slc8_x86-64/ms_gsl/4.0.0-8/include" -I"/sw/slc8_x86-64/HepMC3/3.2.5-18/include" -I"/sw/slc8_x86-64/Vc/1.4.1-25/include" -I"/sw/slc8_x86-64/FairRoot/v18.4.8-37/include" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-2/include/fairmq" -I"/sw/slc8_x86-64/asio/v1.19.1-19/include" -I"/sw/slc8_x86-64/FairLogger/v1.11.1-1/include" -I"/sw/slc8_x86-64/fmt/8.1.1-6/include" -I"/sw/slc8_x86-64/GEANT3/v4-1-21/include/TGeant3" -I"/sw/slc8_x86-64/GEANT4_VMC/v6-1-p1-4/include/g4root" -I"/sw/slc8_x86-64/GEANT4_VMC/v6-1-p1-4/include/geant4vmc" -I"/sw/slc8_x86-64/GEANT4/v11.0.2-1/include" -I"/sw/slc8_x86-64/GEANT4/v11.0.2-1/include/Geant4" -I"/sw/slc8_x86-64/VMC/v2-0-25/include/vmc" -I"/sw/slc8_x86-64/boost/v1.75.0-51/include" -I"/sw/slc8_x86-64/JAliEn-ROOT/0.6.6-102/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Detectors/TRD/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/common/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Constants/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/GPU/Common" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Logger/include" -I"/sw/slc8_x86-64/fmt/8.1.1-6/include" -I"/sw/slc8_x86-64/FairLogger/v1.11.1-1/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/include" -I"/sw/slc8_x86-64/FairRoot/v18.4.8-37/include" -I"/sw/slc8_x86-64/VMC/v2-0-25/include/vmc" -I"/sw/slc8_x86-64/arrow/v9.0.0-alice1-4/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/MathUtils/include" -I"/sw/slc8_x86-64/Vc/1.4.1-25/include" -I"/sw/slc8_x86-64/ms_gsl/4.0.0-8/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/simulation/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Detectors/Common/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/GPU/Utils" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Headers/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/MemoryResources/include" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-2/include/fairmq" -I"/sw/slc8_x86-64/FairMQ/v1.4.55-2/include" -I"/sw/slc8_x86-64/boost/v1.75.0-51/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Utilities/rANS/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Core/include" -I"/sw/slc8_x86-64/Configuration/v2.6.3-42/include" -I"/sw/slc8_x86-64/libInfoLogger/v2.5.1-6/include" -I"/sw/slc8_x86-64/Monitoring/v3.13.2-3/include" -I"/sw/slc8_x86-64/curl/7.70.0-1/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/CCDB/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Utils/include" -I"/sw/slc8_x86-64/libjalienO2/0.1.3-22/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Foundation/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Framework/Foundation/3rdparty/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Utilities/PCG/include" -I"/sw/slc8_x86-64/RapidJSON/v1.1.0-alice2-34/include" -I"/sw/slc8_x86-64/libuv/v1.40.0-25/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/Base/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Reconstruction/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Field/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/DataFormats/Parameters/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/Types/include" -I"/sw/SOURCES/O2/9863-slc8_x86-64/0/Common/SimConfig/include" -I"/sw/slc8_x86-64/TBB/v2021.5.0-17/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/etc/" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/etc//cling" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/etc//cling/plugins/include" -I"/sw/slc8_x86-64/ROOT/v6-26-04-patches-alice1-21/include/" -D__ACLIC__  "/sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit.C" "/sw/BUILD/921c390b408038575f5d92a94a5e177e9bf966f0/O2/compiled_macros//sw/SOURCES/O2/9863-slc8_x86-64/0/Detectors/TRD/calibration/macros/manualCalibFit_C_ACLiC_linkdef.h"' failed!
99% tests passed, 1 tests failed out of 452
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/521c740ee9bb7ca2ccdef72bdbaf9f3247efd7f5/build_O2_o2-cs8/pretty.html).

f3sch committed:
TRD: calibration: Fix suggested command to include correct flag

f3sch committed:
TRD: calibration: Added missed Link Flag to macro test

martenole commented:
> Hi, one should provide the run number via metadata:
> 
> ```
>   auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
>   ...
>   std::map<std::string, std::string> md;
>   md["runNumber"] = std::to_string(runNumber);
>   const auto* grp = ccdbMgr.getSpecific<o2::parameters::GRPECSObject>("GLO/Config/GRPECS", timestamp, md);
> ```
Hi again @shahor02 
But this means we still have to provide a timestamp from somewhere between SOR and EOR, right? Or can this also be fetched automatically somehow, e.g. from bookkeeping or is there a helper in O2 already for this?
Edit: what I would like to achieve is that only the run number needs to be provided to automatically get the SOR and EOR times from CCDB

f3sch committed:
TRD: calibration: using GRPESCObject for retrieving exact run start

Adapted some variable names to conform to code guidelines and added
Xaxis title.

f3sch commented:
Dont merge yet, waiting for @shahor02 to reply to @martenole.

shahor02 commented:
In the macros, where we can use BasicCCDBManager, one can do:
```
auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
...
auto rr = ccdbMgr.getRunDuration(runNumber);
std::map<std::string, std::string> md;
md["runNumber"] = std::to_string(runNumber);
const auto* grp = ccdbMgr.getSpecific<o2::parameters::GRPECSObject>("GLO/Config/GRPECS", (rr.first+rr.second)/2, md);
```
PS: provided the run was registered in http://alice-ccdb.cern.ch/browse/RCT/Info/RunInformation

f3sch committed:
TRD: calibration: plotVdriftExB use runNumber for plots

This makes the macro very easy to use since one must only give
a Run number now and not find out about the start of the Run.
This is also reflected in the README.

f3sch commented:
Works like a charm, thank you @shahor02.
If CI passes now ready for merge once @martenole approves.

alibuild commented:
Error while checking build/O2/o2-cs8 for b82f0f7370e8a65a45600a92da2a25f447cccab5 at 2022-09-21 05:09:
```
## sw/BUILD/O2-latest/log
452/452 Test #408: o2sim_hepmc ..........................................................................***Timeout 400.06 sec
99% tests passed, 1 tests failed out of 452
100% tests passed, 0 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/b82f0f7370e8a65a45600a92da2a25f447cccab5/build_O2_o2-cs8/pretty.html).

alibuild commented:
Error while checking build/O2/o2-dataflow for b82f0f7370e8a65a45600a92da2a25f447cccab5 at 2022-09-20 19:46:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 429
 93/106 Test #107: test_Framework_test_SingleDataSource ....................***Failed    2.81 sec
[ERROR] pid 25302 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] SEVERE: Device internal-dpl-injected-dummy-sink (25302) returned with 1
99% tests passed, 1 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/b82f0f7370e8a65a45600a92da2a25f447cccab5/build_O2_o2-dataflow/pretty.html).

alibuild commented:
Error while checking build/O2/o2 for b82f0f7370e8a65a45600a92da2a25f447cccab5 at 2022-09-20 21:17:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 452
 76/106 Test  #90: test_Framework_test_CallbackService .....................***Failed    5.42 sec
[ERROR] pid 26176 (dest) crashed with 1
[ERROR] SEVERE: Device dest (26176) returned with 1
 86/106 Test #100: test_Framework_test_SimpleDataProcessingDevice01 ........***Failed    2.24 sec
[ERROR] pid 26658 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] SEVERE: Device internal-dpl-injected-dummy-sink (26658) returned with 1
98% tests passed, 2 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/b82f0f7370e8a65a45600a92da2a25f447cccab5/build_O2_o2/pretty.html).

martenole commented on Detectors/TRD/calibration/macros/plotVdriftExB.C:
```diff
@@ -0,0 +1,133 @@
+void ccdbDownload(unsigned int runNumber, std::string ccdb, timePoint timeOffset)
+{
+  auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbMgr.setURL("http://alice-ccdb.cern.ch/");
```
probably you should set the URL to `ccdb` here

f3sch replied:
Didn't you tell me the GRPECSObject are only alice-ccdb not anywhere
else?
Maybe adding a second parameter for the ccdb for GRPECSObjects is better?

martenole replied:
Yes. Ah, ok, you are changing the URL below to `ccdb` to get the TRD object from the specified path and only the GRP always from alice-ccdb. This I missed. Then this is fine.

martenole commented on Detectors/TRD/calibration/macros/plotVdriftExB.C:
```diff
@@ -0,0 +1,133 @@
+// The times must be given in unix epoch ms format.
+// The default Run is 523677 for the test ccdb.
+// The offset for the next calibration is 15 Minutes.
+void plotVdriftExB(unsigned int runNumber = 523677, std::string ccdb = "http://ccdb-test.cern.ch:8080", timePoint timeOffset = 899991)
```
Why do you not take 15 minutes (900000) for the `timeOffset`? Also something like `queryInterval` might be more intuitive, but this is up to you

f3sch replied:
I do not recall the reason for this exactly but I think in the ccdb the offsets are like this.
But changing this is works just as well.
Renaming is probably better.
_There are only two hard things in Computer Science: cache invalidation, naming things and off by one errors._

f3sch committed:
TRD: calibration: changed variable name and fixed interval time to 15min

alibuild commented:
Error while checking build/O2/o2-dataflow for d083df43542d19ddfbddf4af64a11fc8f1430c27 at 2022-09-22 04:44:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 429
 86/106 Test #100: test_Framework_test_SimpleDataProcessingDevice01 ........***Failed    2.24 sec
[ERROR] pid 11184 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] SEVERE: Device internal-dpl-injected-dummy-sink (11184) returned with 1
99% tests passed, 1 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/d083df43542d19ddfbddf4af64a11fc8f1430c27/build_O2_o2-dataflow/pretty.html).
